server/app.py

```diff
@@ -1,11 +1,21 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+
+import os
+from dotenv import load_dotenv
+
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
+from config import Config
 
-
+load_dotenv(".env")
```

Comment:
I don't think we need this piece of code, since pipenv will already load .env (https://docs.pipenv.org/en/latest/advanced/ - see Automatic loading of .env)

Reply:
Removed line. Thanks for the pointer!